src/api/authAPI.ts

```diff
@@ -0,0 +1,51 @@
+import { FetchSigninInfoTypes } from '@/types';
+
+const BASE_URL = process.env.NEXT_PUBLIC_KEYDEUK_API_BASE_URL;
+
+export const checkEmailDuplication = async (emailValue: string) => {
+  const url = `${BASE_URL}/api/v1/users/check/email?email=${emailValue}`;
+  const response = await fetch(url);
+  const data = await response.json();
+  return data;
+};
+
+export const checkNicknameDuplication = async (nickname: string) => {
+  const url = `${BASE_URL}/api/v1/users/check/nickname?nickname=${nickname}`;
+  const response = await fetch(url);
+  const data = await response.json();
+  return data;
+};
+
+export const postSignup = async (formData: FormData) => {
+  const url = `${BASE_URL}/api/v1/users`;
+
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+      headers: {
+        accept: 'application/json',
+      },
+      body: formData,
+    });
```

Comment:
```suggestion
      headers: {
        'Content-Type': 'application/json',
      },
      body: JSON.stringify(formData),
    });
```
이렇게 보내야하지 않나요? 처음보는 방식이네유

Reply:
보내는 바디 형식이 form-data 형식이라 저렇게 보냈어요!! 혹시 몰라서 Json.stringify로 시도해봤는데 에러 뜨네욤,,

Reply:
https://damagucci-juice.tistory.com/entry/HTTP-Accept-type-header-%EC%99%80-Content-Type-header-%EC%9D%98-%EC%B0%A8%EC%9D%B4%EC%A0%90
이것도 일반적인 사용은 아닌거같아요

Reply:
아 네넵 그러네요 header 에 json이 들어가면 안되겠네요! 이 부분은 Formdata 타입으로 맞춰놓겠습니다!